### PR TITLE
Fix roles

### PR DESCRIPTION
**Pull request by JKL98ISR**

### CODE REVIEW

File: frontend/src/api/generated.ts [hunk 1]
```diff
@@ -186,7 +186,7 @@ export type WebhookKind = typeof WebhookKind[keyof typeof WebhookKind];
 
 // eslint-disable-next-line @typescript-eslint/no-redeclare
 export const WebhookKind = {
-  STANDART: 'STANDART',
+  STANDARD: 'STANDARD',
```
Comment (yromanyshyn) [inline frontend/src/api/generated.ts:189]: oh, yeap, my bad, I forget about the types on the frontend, thanks!

File: backend/deepchecks_monitoring/ee/api/v1/members.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ async def update_user_role(roles_schema: RoleUpdateSchema,
     if user.organization_id != current_user.organization_id:
         raise BadRequest("User doesn't exists in your organization.")
 
-    roles: t.List[Role] = await session.scalars(sa.select(Role).where(Role.user_id == user_id))
+    roles: t.List[Role] = (await session.execute(sa.select(Role).where(Role.user_id == user_id))).scalars().all()
```
Comment (yromanyshyn) [inline backend/deepchecks_monitoring/ee/api/v1/members.py:57]: why? `session.scalars` does exactly the same thing, does not it?

**Review by yromanyshyn — APPROVED** [review]

File: backend/deepchecks_monitoring/ee/api/v1/members.py (hunk 2 above)
Comment (yromanyshyn) [reply backend/deepchecks_monitoring/ee/api/v1/members.py:57]: `session.scalars` body
image

Comment (JKL98ISR) [reply backend/deepchecks_monitoring/ee/api/v1/members.py:57]: it doesn't, it what made the bug

**Pull request merged**